src/cmd/health.js

```diff
@@ -334,6 +346,11 @@ async get_storage_status(config_root, type, all_details) {
       const { data } = await nb_native().fs.readFile(fs_context, config_file_path);
       config_data = JSON.parse(data.toString());
       storage_path = type === 'bucket' ? config_data.path : config_data.nsfs_account_config.new_buckets_path;
+      // check for access in account new_buckets_path dir
+      if (type === 'account') {
+        await nb_native().fs.stat(this.get_account_fs_context(config_data.nsfs_account_config.uid,
```

Comment:
stat() does not require permissions to the file itself, accoridng to **https://linux.die.net/man/2/stat** -
```
No permissions are required on the file itself, but-in the case of stat() and lstat() - execute (search) permission is required on all of the directories in path that lead to the file.
```

use checkAccess instead -
await nb_native().fs.checkAccess(fs_context, storage_path);

Reply:
@romayalon updated with `checkAccess()`